<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pt_slider_dialog</class>
 <widget class="QDialog" name="pt_slider_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PT Slider: </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QGroupBox" name="pressure_group">
     <property name="title">
      <string>Pressure</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="pressure_slider_range_label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure Slider Range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Slider Range:</string>
        </property>
        <property name="buddy">
         <cstring>pressure_slider_range</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSlider" name="pressure_slider">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="pressure_slider_range">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure Slider Range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> GPa</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ScientificDoubleSpinBox" name="pressure">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> GPa</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="parameters_group">
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="k0p_label">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;dK/dP&lt;/span&gt; - the derivative of the bulk modulus with respect to pressure&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;K'&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;0 &lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>k0p</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="k0_label">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bulk Modulus (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;K&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;0 &lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>k0</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ScientificDoubleSpinBox" name="k0p">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;dK/dP&lt;/span&gt; - the derivative of the bulk modulus with respect to pressure&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScientificDoubleSpinBox" name="k0">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bulk Modulus (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> GPa</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="temperature_group">
     <property name="title">
      <string>Temperature</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="7" column="0" colspan="2">
       <widget class="QSlider" name="temperature_slider">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="ScientificDoubleSpinBox" name="temperature_slider_range">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure Slider Range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> K</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="ScientificDoubleSpinBox" name="temperature">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> K</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>298.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="temperature_slider_range_label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure Slider Range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Slider Range:</string>
        </property>
        <property name="buddy">
         <cstring>temperature_slider_range</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="4" column="0">
          <widget class="QLabel" name="dalpha_t_dt_label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;d&lt;span style=&quot; font-family:'sans-serif'; font-style:italic; color:#202122;&quot;&gt;α&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;T &lt;/span&gt;/d&lt;span style=&quot; font-style:italic;&quot;&gt;T&lt;/span&gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>dalpha_t_dt</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="ScientificDoubleSpinBox" name="dk0pdt">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> K^-1</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="alpha_t_label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'sans-serif'; font-style:italic; color:#202122;&quot;&gt;α&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;T &lt;/span&gt;&lt;span style=&quot; font-family:'-apple-system','BlinkMacSystemFont','Helvetica Neue','Helvetica','Arial','sans-serif'; font-size:22px; color:#404040;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>alpha_t</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="ScientificDoubleSpinBox" name="dalpha_t_dt">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> K^-2</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="ScientificDoubleSpinBox" name="alpha_t">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> K^-1</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="ScientificDoubleSpinBox" name="dk0dt">
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> GPa/K</string>
           </property>
           <property name="decimals">
            <number>8</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="dk0pdt_label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;d&lt;span style=&quot; font-style:italic;&quot;&gt;K'&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;0 &lt;/span&gt;/d&lt;span style=&quot; font-style:italic;&quot;&gt;T &lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>dk0pdt</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="dk0dt_label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;d&lt;span style=&quot; font-style:italic;&quot;&gt;K&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;0 &lt;/span&gt;/ d&lt;span style=&quot; font-style:italic;&quot;&gt;T&lt;/span&gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="buddy">
            <cstring>dk0dt</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="read_from_jcpds_button">
     <property name="text">
      <string>Read Parameters from JCPDS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="ambient_structure">
     <property name="title">
      <string>Ambient Structure</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <widget class="QLabel" name="ambient_b_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>b:</string>
        </property>
        <property name="buddy">
         <cstring>ambient_b</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="ambient_beta_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>β:</string>
        </property>
        <property name="buddy">
         <cstring>ambient_beta</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="ScientificDoubleSpinBox" name="ambient_c">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ScientificDoubleSpinBox" name="ambient_a">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ambient_a_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>a:</string>
        </property>
        <property name="buddy">
         <cstring>ambient_a</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="ScientificDoubleSpinBox" name="ambient_alpha">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ambient_c_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>c:</string>
        </property>
        <property name="buddy">
         <cstring>ambient_c</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="ambient_gamma_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>γ:</string>
        </property>
        <property name="buddy">
         <cstring>ambient_gamma</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="ambient_alpha_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>α:</string>
        </property>
        <property name="buddy">
         <cstring>ambient_alpha</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="ambient_b">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="ScientificDoubleSpinBox" name="ambient_gamma">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="ScientificDoubleSpinBox" name="ambient_beta">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000010000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QPushButton" name="set_ambient_structure_button">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the current lattice parameters to be the ambient structure.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Set Ambient Structure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>read_from_jcpds_button</tabstop>
  <tabstop>k0</tabstop>
  <tabstop>k0p</tabstop>
  <tabstop>pressure_slider_range</tabstop>
  <tabstop>pressure_slider</tabstop>
  <tabstop>pressure</tabstop>
  <tabstop>dk0dt</tabstop>
  <tabstop>dk0pdt</tabstop>
  <tabstop>alpha_t</tabstop>
  <tabstop>dalpha_t_dt</tabstop>
  <tabstop>temperature_slider_range</tabstop>
  <tabstop>temperature_slider</tabstop>
  <tabstop>temperature</tabstop>
  <tabstop>ambient_a</tabstop>
  <tabstop>ambient_alpha</tabstop>
  <tabstop>ambient_b</tabstop>
  <tabstop>ambient_beta</tabstop>
  <tabstop>ambient_c</tabstop>
  <tabstop>ambient_gamma</tabstop>
  <tabstop>set_ambient_structure_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
